packages/web_benchmarks/lib/client.dart

```diff
@@ -127,33 +130,32 @@ void _fallbackToManual(String error) {
         ${_benchmarks.keys.map((String name) => '<li><button id="$name">$name</button></li>').join('\n')}
       </ul>
     </div>
-  ''',
-      validator: html.NodeValidatorBuilder()
-        ..allowHtml5()
-        ..allowInlineStyles());
+  ''');
 
```

Comment:
is this safe to remove?

Reply:
Given that we generate 100% of the HTML embedded, I believe yes!